### PR TITLE
Fixed #32662 -- Refactored a generator out of SQLCompiler.get_order_by().

### PR DESCRIPTION
**Pull request by cjerdonek**
https://code.djangoproject.com/ticket/32662

### CODE REVIEW

File: django/db/models/sql/compiler.py [hunk 1]
```diff
@@ -290,11 +282,10 @@ def get_order_by(self):
         else:
             ordering = []
         if self.query.standard_ordering:
-            asc, desc = ORDER_DIR['ASC']
+            asc, _desc = ORDER_DIR['ASC']
         else:
-            asc, desc = ORDER_DIR['DESC']
+            asc, _desc = ORDER_DIR['DESC']
```
Comment (felixxm) [inline django/db/models/sql/compiler.py:287]: `asc` is misleading, because it's basically a default ordering. Renamed to the `default_order`.

**Review by felixxm — COMMENTED** [review]
@cjerdonek Thanks for this patch :+1: 

I think we can squash commits.

**Comment by cjerdonek** [comment]
Thanks! 

> I think we can squash commits.

And for sure, those were just for your benefit.

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Pull request merged**